test/s3-benchmarks/.scripts/create_benchmark_files

```diff
@@ -1,14 +1,14 @@
-head -c 1B </dev/urandom >/dev/shm/1B
-head -c 8388607B </dev/urandom >/dev/shm/8MB-1
-head -c 8388609B </dev/urandom >/dev/shm/8MB+1
-head -c 128M </dev/urandom >/dev/shm/128MB
+head -c 1b </dev/urandom >/dev/shm/1B
```

Comment:
Can we name this file 1 KB? I thought it was 1 byte based on the naming.

Reply:
Maybe with all of these inconsistencies across platforms, we should just do the math ourselves?

1024 for 1 KB
`$((30*1024*1024*1024))` for 30 GB, etc.

The EC2 instance I checked doesn't support B, for example.